packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -95,7 +95,27 @@ const mapAttachmentType = (attachmentType: AttachmentType): FileMetadataAttachme
   return 'unknown';
 };
 
-/* @conditional-compile-remove(file-sharing) @conditional-compile-remove(teams-inline-images) */
+const processTeamsImageContent = (message: ChatMessageWithStatus): string | undefined => {
+  /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+  if (sanitizedMessageContentType(message.type).includes('html') && message.content?.attachments) {
+    let teamsImageHtmlContent = '';
+    const attachments: ChatAttachment[] = message.content?.attachments;
+    attachments.forEach((attachment) => {
+      if (attachment.attachmentType === 'teamsImage') {
+        teamsImageHtmlContent += `\r\n<p><img alt="image" src="" itemscope="${attachment.contentType}" id="${attachment.id}"></p>`;
+      }
+    });
+    if (teamsImageHtmlContent === '') {
+      return message.content?.message;
+    }
+    const content = (message.content?.message ?? '') + teamsImageHtmlContent;
+    const trialingEmptyLine = '\r\n<p>&nbsp;</p>';
+    return content.endsWith(trialingEmptyLine) ? content.slice(0, content.lastIndexOf(trialingEmptyLine)) : content;
```

Comment:
what is this for?

Reply:
When Teams send inline image and attachment, it will have duplicated 'newLine' `<p>` tags this is to trim trailing new lines

Reply:
ah i see, if this is the case, do we want to wrap this into some if check so it only applies when we have both inline images and attachments